### PR TITLE
Game loop

### PR DESCRIPTION
**Pull request by scottjamessmith**
Code for the barebones of the proposed class organization:

Screenshot 2021-02-22 164429

### CODE REVIEW

File: Assets/Scripts/Player.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+/*
+    Abstract class to be inherited by a local player, networked player, client player?, and AI player
+ */
+
+public /* abstract */ class Player : MonoBehaviour
```
Comment (neoJINXD) [inline Assets/Scripts/Player.cs:8]: is there a reason abstract is still commented?

**Review by neoJINXD — APPROVED** [review]
Other than that one comment, looks good

File: Assets/Scripts/Player.cs (hunk 1 above)
Comment (scottjamessmith) [reply Assets/Scripts/Player.cs:8]: ye cause in the gamemanager right now it just has two player classes added at run time for super basic testing. if you run the main scene you'll see two players attached and whichever's turn it has an editor button to finish their turn

Comment (neoJINXD) [reply Assets/Scripts/Player.cs:8]: got it, so keeping it not abstract until we make the other player classes so that the demo you have now works

**Comment by neoJINXD** [comment]
Good to merge then

**Comment by JonathanSolvesProblems** [comment]
LGTM! J'approve

**Pull request merged**